### PR TITLE
41 Navbar

### PR DESCRIPTION
**Pull request by SouchetJulie**
## New
 * [client] Add a navbar present on all pages. It shows different links if the user is logged in...
image
... or not.
image
 * [client] Add sidebar
 * [client] Make both navbar and sidebar responsive and accessible

 * [internal] Add hook `useUser` to get current user client-side

### CODE REVIEW

**Review by alland07 — DISMISSED** [review]

**Review by alland07 — CHANGES_REQUESTED** [review]
image
 Page de connexion, il faut enlever l'ascenseur @SouchetJulie

File: src/client/components/signIn_signUp/LoginForm.component.tsx [hunk 1]
```diff
@@ -1,91 +1,82 @@
...
       }
     } else {
       // Add
       const success: boolean = false;
-      const message = (
-        <span>
-          Veuillez remplir le formulaire
-        </span>
-      );
-      dispatch(addAlert({ message, success}));
```
Comment (alland07) [inline src/client/components/signIn_signUp/LoginForm.component.tsx:88]: Tu as oublier de modifier ici

**Review by alland07 — CHANGES_REQUESTED** [review]
Dans l'ensemble c'est bon pour moi, regarde mon com ^^

File: src/client/components/signIn_signUp/LoginForm.component.tsx (hunk 1 above)
Comment (SouchetJulie) [reply src/client/components/signIn_signUp/LoginForm.component.tsx:88]: Done 👍

**Review by alland07 — CHANGES_REQUESTED** [review]
image

**Review by alland07 — APPROVED** [review]

**Pull request merged**